core/workspace_comment_svg.js

```diff
@@ -49,6 +49,21 @@ goog.require('Blockly.WorkspaceComment');
  */
 Blockly.WorkspaceCommentSvg = function(workspace, content, height, width,
     opt_id) {
+
+  /**
+   * Mouse up event data.
+   * @type {?Blockly.EventData}
+   * @private
+   */
+  this.onMouseUpWrapper_ = null;
+
+  /**
+   * Mouse move event data.
+   * @type {?Blockly.EventData}
+   * @private
+   */
+  this.onMouseMoveWrapper_ = null;
+
```

Comment:
Looks like these were just added, but not set to anything.

Reply:
They're [used inside the workspace_comment_render_svg](https://github.com/google/blockly/blob/ba6dfd812552e0823ed75f3dc5133a267cc6fbc9/core/workspace_comment_render_svg.js#L266) file, so I believe that means they need to be declared here. Honestly I'm not sure though.

Reply:
Sounds good.